wavefront/server/background_jobs/rag_ingestion/rag_ingestion/main.py

```
@@ -15,11 +15,13 @@ def main():
     cache_manager = CacheManager(namespace='rag')
     encryption_service = None
     if (
-        (CLOUD_PROVIDER == 'aws' and os.getenv('AWS_KMS_ARN') is not None)
+        (CLOUD_PROVIDER == 'aws' and os.getenv('AWS_KMS_ARN'))
         or CLOUD_PROVIDER == 'gcp'
-        and (
-            os.getenv('GCP_KMS_KEY_RING') is not None
-            and os.getenv('GCP_KMS_CRYPTO_KEY') is not None
+        and (os.getenv('GCP_KMS_KEY_RING') and os.getenv('GCP_KMS_CRYPTO_KEY'))
```

Ruff (0.15.6)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)